examples/seismic/self_adjoint/time_blocking.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>probably it needs an NBVAL_IGNORE_OUTPUT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
agree!